core/app/helpers/spree/base_helper.rb

```diff
@@ -106,8 +106,8 @@ def taxons_tree(root_taxon, current_taxon, max_level = 1)
       end
     end
 
-    def available_countries
-      checkout_zone = Zone.find_by(name: Spree::Config[:checkout_zone])
+    def available_countries(zone_name = Spree::Config[:checkout_zone])
```

Comment:
I would like to have a more meaningful parameter here. Just passing `nil` to restrict the countries to a specific zone seams unintuitive to me.

Maybe something like?

```rb
def available_countries(restrict_to_zone: Spree::Config[:checkout_zone])
```